downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -87,6 +87,13 @@ enum class VcsHost(
                     }
                 }
             }
+
+        override fun getArchiveDownloadUrlInternal(vcsInfo: VcsInfo): String? {
```

Comment:
I believe all VcsHost-related changes should be moved to a separate preparing commit as they are independent of the Carthage changes.

Reply:
Yes, they are independent of the Carthage change.
I thought about moving these changes to a separate commit but decided against it because all new methods would be unused (well, only used by unit tests).
Anyway: Because they are usable without the Carthage implementation I will move them to a separate commit.